permissions-request/idlharness.any.js

```diff
@@ -0,0 +1,27 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+idl_test(
+  ['permissions-request'],
+  ['permissions', 'html', 'dom'],
+  async idl_array => {
+    // Attempt to make a permissions object available via navigator.
+    try {
+      await navigator.permissions.query({ name: "geolocation" });
```

Comment:
Does this actually change the outcome of the test? The resolved value would be a https://w3c.github.io/permissions/#permissionstatus instance, but this wouldn't be the test to include it. `navigator.permissions` is a https://w3c.github.io/permissions/#permissions instance without doing anything, right?

Reply:
You appear to be correct. I copy/pasted from another permissions idlharness test, without actually checking if this was necessary.